torchvision/utils.py

```diff
@@ -242,8 +242,9 @@ def draw_segmentation_masks(
         masks (Tensor): Tensor of shape (num_masks, H, W) or (H, W) and dtype bool.
         alpha (float): Float number between 0 and 1 denoting the transparency of the masks.
             0 means full transparency, 1 means no transparency.
-        colors (list or None): List containing the colors of the masks. The colors can
-            be represented as PIL strings e.g. "red" or "#FF00FF", or as RGB tuples e.g. ``(240, 10, 157)``.
+        colors: Union[List[Union[str, Tuple[int, int, int]]], str, Tuple[int, int, int]]: List containing the colors
```

Comment:
We should avoid using the annotation style in the docstrings, as they can be very hard to read and understand, especially this one. In this case we can keep things simple along with a good explanation in the description as you already wrote:
 
```suggestion
        colors (color or list of colors, optional): List containing the colors
```

Reply:
We don't write this way in draw_bounding_boxes.
Do you want me to update there as well?

Reply:
it would be nice, thanks :)